gqlerrors/error.go

```diff
@@ -38,6 +42,9 @@ func NewError(message string, nodes []ast.Node, stack string, source *source.Sou
 	}
 	if len(positions) == 0 && len(nodes) > 0 {
 		for _, node := range nodes {
+			if node == nil || reflect.ValueOf(node).IsNil() {
```

Comment:
If you're asking because you think it's holding up the merge, it's not. I was just posing the question.